addons/image-uploader/userscript.js

```diff
@@ -1,4 +1,4 @@
-import textFieldEdit from "../../libraries/text-field-edit.js"; //used for editing the forum text box without messing with the edit history
+﻿import textFieldEdit from "../../libraries/text-field-edit.js"; //used for editing the forum text box without messing with the edit history
```

Comment:
Why was this file edited?

Reply:
> 
> 
> Also, somehow something really went wrong because I didn't want to add that fix for the image uploader but somehow I did.
> Well, it still just improves everything a tiny bit (now it says that you're able to drop - that's a lie but before it said that you're unable to drop what is even more unwanted), drag and drop still doesn't work but now you get a different error.



> 
> 
> Well ... that problematig drag&drop line is the one where @JeffaloBob said that he has no clue if it works with firefox.
> Line 107 in case someone is interested.

Reply:
Oh ok.

Reply:
Um ... should I reverse that? I mean, undoing a """""fix""""" is kinda stupid.

Reply:
Not sure. Jeffalo made the addon so he knows best. Having both edits in the same PR was a bad idea.

Reply:
Indeed. As I said - I did something really wrong because I don't know how to use GitHub Desktop. 
I can undo it.

Reply:
So should I? @JeffaloBob ?

Reply:
yeah i think you should undo it, i'll work on firefox support when i redo it

Reply:
Okay. Keep that ondragover thing in mind.

Reply:
@JeffaloBob now that's done.